k-distribution/tests/regression-new/imp-haskell/Makefile

```diff
@@ -2,7 +2,7 @@ DEF=imp
 EXT=imp
 TESTDIR=.
 KOMPILE_BACKEND=haskell
-KOMPILE_FLAGS=--post-process 'cat' --no-haskell-binary
+KOMPILE_FLAGS=--post-process 'cat'
```

Comment:
This should not be set ad-hoc per test but instead added to the global `KOMPILE_FLAGS+=...` directive in the .mak files